data_steward/resource_files/aou_vocab/concept.csv

```diff
@@ -10,6 +10,7 @@ concept_id	concept_name	domain_id	vocabulary_id	concept_class_id	standard_concep
 2000000008	More than one population	Observation	AoU_General	Answer	S	WhatRaceEthnicity_GeneralizedMultPopulations	2017-05-17	2099-12-31	
 2000000009	Not male, not female, prefer not to answer, or skipped	Observation	AoU_General	Answer	S	SexAtBirth_GeneralizedDiffSex	2017-05-17	2099-12-31	
 2000000010	Response removed due to invalid value	Observation	AoU_General	Answer	S	AoUDRC_ResponseRemoval	2017-05-17	2099-12-31	
-2000000011	State Information Suppressed for Privacy	Observation	AoU_General	Answer	S	AoUDRC_GeneralizedState	2017-05-17	2099-12-31	
+2000000011	State Information Suppressed for Privacy	Observation	AoU_General	Answer	S	AoUDRC_GeneralizedState	2017-05-17	2099-12-31
+2000000013  11 or more household members    Observation AoU_General Answer  S   HouseSize_11+   2017-05-17  2099-12-31
```

Comment:
```suggestion
2000000013  11 or more household members    Observation AoU_General Answer  S   HouseSize_11+   2017-05-17  2099-12-31
```


Spaces were used instead of tabs in this line and there is one empty column after the end date a tab needs to be added at the end.

Reply:
@ksdkalluri, what's the best way to edit this file? I originally did it through PyCharm and used tabs but they came out as spaces instead :\